src/render.ts

```diff
@@ -105,8 +105,9 @@ export async function render(
     case "Vue": {
       const Vue = await require("vue");
       const app = Vue.createApp({
-        setup: () => () => Vue.h(storyResult)
+        setup: () => () => Vue.h(storyResult),
       });
+      storyResult.appDecorator?.(app);
```

Comment:
That was part of my previous change when the story provided the `app` instance prepared properly.

I'm afraid that setting up the `h(story)` before any plugin installed will create an issue. I'm not sure all plugins will "backprocess" what is in the root before installing.